subproject1/code/descriptive_analysis.R

```diff
@@ -1,7 +1,47 @@
 here::i_am(
-"subproject1/code/data_clean.R"
+  "subproject1/code/descriptive_analysis.R"
 )
 #Please fill this section with data cleaning 
+library(dplyr)
+library(tidyr)
+library(ggplot2)
+
+# Create the contingency table
```

Comment:
it will be better if you read the data_clean.rds first before doing next step. If someone just want to run this file without cleaning data, the R studio won't store the content of cleand data which means they will find "data_clean" undefined.